Marlin/Configuration.h

```diff
@@ -2124,3 +2124,8 @@
 
 // Allow servo angle to be edited and saved to EEPROM
 //#define EDITABLE_SERVO_ANGLES
+
+// Structured configuration, see #13752
+#if __has_include("User_Configuration.h")
+   #include "User_Configuration.h"
+#endif
```

Comment:
Wow, I had no idea there was a `__has_include` in C++. Mind blown.

Reply:
there is also `__has_include_next` https://gcc.gnu.org/gcc-5/changes.html
```
This function-like macro takes a single file name string argument that is the name of an include file. It is like __has_include except that it looks for the second instance of the given file found in the include paths. It evaluates to 1 if the second instance of the file can be found using the include paths, or 0 otherwise
```